<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A*</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/application-xp.png</normaloff>:/icons/application-xp.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="renderHints">
       <set>QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="clearAction"/>
   <addaction name="separator"/>
   <addaction name="selectAction"/>
   <addaction name="addVerticleAction"/>
   <addaction name="removeAction"/>
   <addaction name="separator"/>
   <addaction name="startAction"/>
   <addaction name="resetAction"/>
  </widget>
  <action name="addVerticleAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/record_gray.png</normaloff>:/icons/record_gray.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj wierzchołek</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="removeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/cross.png</normaloff>:/icons/cross.png</iconset>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="startAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/resultset_next.png</normaloff>:/icons/resultset_next.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="resetAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/decline.png</normaloff>:/icons/decline.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="selectAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/cursor.png</normaloff>:/icons/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Wybieranie / Przesuwanie</string>
   </property>
   <property name="toolTip">
    <string>Wybieranie / Przesuwanie</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="loadAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Otwórz...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/disk.png</normaloff>:/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/page_white.png</normaloff>:/icons/page_white.png</iconset>
   </property>
   <property name="text">
    <string>Wyczyść</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addVerticleAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>startAlgorithm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetAlgorithm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setMode()</slot>
  <slot>startAlgorithm()</slot>
  <slot>resetAlgorithm()</slot>
  <slot>load()</slot>
  <slot>save()</slot>
  <slot>clear()</slot>
 </slots>
</ui>
